src/frontend/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
I'll make this workflow slightly more generic & add it to hotosm/gh-workflows =)

Then we can use the same workflow across projects - thanks!

I will move this into the pr_test_frontend.yml workflow, so it only runs on frontend changes 👍